arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -749,18 +743,21 @@ sealed class Option<out A> : OptionOf<A> {
       is None -> null
     }
 
+  @Deprecated(TraverseDeprecation)
```

Comment:
Idem for this file.

https://github.com/arrow-kt/arrow-core/blob/0.11.0/arrow-core-data/src/main/kotlin/arrow/core/Option.kt